src/Keboola/DbExtractor/Extractor/Snowflake.php

```diff
@@ -220,13 +220,16 @@ private function exportAndDownload(array $table): int
             sprintf('Copy Command: %s failed with message', $copyCommand)
         );
 
-        if (count($res) > 0 && (int) $res[0]['rows_unloaded'] === 0) {
+        $rowCount = 0;
```

Comment:
OJ sorry, vcera som to prehliadol.
Nechces to zapisat jednoduchsie/prehladnejsie? `$rowCount = (int) ($res[0]['rows_unloaded'] ?? 0);`

See: http://sandbox.onlinephpfunctions.com/code/4aa679e15052307877b817dd2157e033f3b34cd4

Reply:
ok to by šlo - je to tam